### PR TITLE
Add base64Encode action

### PR DESCRIPTION
**Pull request by bachya**
**Checks**

- [x] I've checked there are no linting errors.
- [x] I've checked the code is still at 100% test coverage.

**Added Actions (if relevant)**

- `Base 64 Encode`

**Are you happy to be listed as a contributor on [Shortcuts.fun](https://shortcuts.fun)?**

Yes

### CODE REVIEW

**Comment by bachya** [comment]
I don't understand why coverage is decreasing on this one?

**Comment by xAlien95** [comment]
> I don't understand why coverage is decreasing on this one?

The failing coverage test is probably to the absence of a test with no parameters.
Check [`actions/text.spec.ts#L11`](https://github.com/joshfarrant/shortcuts-js/blob/master/__tests__/actions/text.spec.ts#L11) for a reference.

**Comment by bachya** [comment]
Great call, @xAlien95; thank you.

File: src/actions/base64Encode.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { withActionOutput } from '../utils';
+
+import WFBase64LineBreakMode from '../interfaces/WF/WFBase64LineBreakMode';
+import WFEncodeMode from '../interfaces/WF/WFEncodeMode';
+import WFSerialization from '../interfaces/WF/WFSerialization';
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Base 64 Encode Action. Base 64-encode content.
```
Comment (joshfarrant) [inline src/actions/base64Encode.ts]: Can this be changed to match the action description in the app please? 🙂

**Pull request merged**